### PR TITLE
[FIX] Update help description for "use" and "--framework-version"

### PR DESCRIPTION
**Pull request by matz3**
Updates according to new features via https://github.com/SAP/ui5-project/pull/649

JIRA: CPOUI5FOUNDATION-722

### CODE REVIEW

**Review by flovogt — APPROVED** [review]

File: lib/cli/commands/build.js [hunk 1]
```diff
@@ -101,7 +101,8 @@ build.builder = function(cli) {
 			array: true
 		})
 		.option("framework-version", {
-			describe: "Overrides the framework version defined by the project",
+			describe: "Overrides the framework version defined by the project. " +
+				"Takes the same values as the version part of \"ui5 use\"",
```
[the remark this answers is not in the document]
Comment (matz3) [reply lib/cli/commands/build.js:105]: Yes, it's just one value, thanks!

Comment (KlattG) [inline lib/cli/commands/build.js:105]:
```suggestion
				"Takes the same value as the version part of \"ui5 use\"",
```

Comment (matz3) [reply lib/cli/commands/build.js]: I did a amend of the existing commit.

**Review by flovogt — APPROVED** [review]

**Review by KlattG — APPROVED** [review]
LGTM

**Comment by matz3** [comment]
Thanks 👍🏻

**Pull request merged**